### PR TITLE
Renders collection names as links

### LINKED ISSUE #334: Curator dashboard should link to the collections they curate
On the Curator's dashboard we show which collection the user curates, but do not give them a way to access that collection.
Screen Shot 2022-08-24 at 2 49 01 PM

This will be complete when
- [ ] The collection list is a collection of links instead of just plain text

### PR DESCRIPTION
**Pull request by hectorcorrea**
collection_name_as_link

Closes #334

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -22,7 +22,10 @@
 <% if @user.super_admin? %>
   <!-- not sure we want to display the collections here, it's pointless -->
 <% elsif @user.curator? %>
-  <p>Curator for: <%= @user.admin_collections.map(&:title).to_sentence %></p>
+  <p>Curator for:
+    <%= @user.admin_collections.map do |collection|
+      link_to(collection.title, collection_url(collection))
```
Comment (carolyncole) [inline app/views/users/show.html.erb:27]: @hectorcorrea is there a reason not to utilize relative paths?
```suggestion
      link_to(collection.title, collection_path(collection))
```

File: spec/spec_helper.rb [hunk 2]
```diff
@@ -111,3 +111,7 @@
   #   # as the one that triggered the failure.
   #   Kernel.srand config.seed
 end
+
+def capybara_root_url
```
Comment (carolyncole) [inline spec/spec_helper.rb:115]: I believe if we utilize relative urls this would not be needed

File: spec/system/user_show_spec.rb [hunk 3]
```diff
@@ -12,5 +13,13 @@
       visit user_path(user_admin)
       expect(page.html.include?(sort_value)).to be true
     end
+
+    it "renders the collection names as links", js: true do
+      sign_in moderator_user
+      visit user_path(moderator_user)
+      pppl_url = capybara_root_url + collection_path(Collection.plasma_laboratory)
```
Comment (carolyncole) [inline spec/system/user_show_spec.rb:20]:
```suggestion
      pppl_url = collection_path(Collection.plasma_laboratory)
```

**Review by carolyncole — CHANGES_REQUESTED** [review]
I'm thinking utilizing relative path would make this feature simpler in the long run.

**Comment by hectorcorrea** [comment]
@carolyncole good idea, I switched to use relative paths.

**Review by carolyncole — APPROVED** [review]

**Pull request merged**